samples/adminconfig/infrastructure.json

```
@@ -1,45 +1,65 @@
 {
+    "metrics": [
+        {
+            "name": "cost",
+            "type": "numeric",
+            "units": "US Dollar per TB per month",
+            "scale": 
+            {
+                "min": 0,
+                "max": 500
+            }
+        },
+        {
+            "name": "distance",
+            "type": "numeric",
+            "units": "km",
+            "scale":
+            {
+                "min": 0,
+                "max": 20000
+            }
+        }
+    ],
     "infrastructure": [
         {
             "attribute": "storage-cost",
             "description": "neverland object store",
+            "metricsName": "cost",
```

```suggestion
            "metricName": "cost",
```
Should be singular, because it's only one metric we are using (but verify with Sima)

---

Changed to metricName

---

Agree. singular